<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewfileDialog</class>
 <widget class="QDialog" name="NewfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New File Properties</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinCinF">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>95</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinFirst">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Characters in Font</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Encoding</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>63</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinHeight">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>14</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinWidth">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ascent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinAscent">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>91</width>
     <height>75</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Pixel Depth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio1BPP">
      <property name="text">
       <string>1BPP</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio2BPP">
      <property name="text">
       <string>2BPP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>newfont</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
